src/components/SearchTracePage/TraceSearchForm.js

```diff
@@ -246,6 +253,7 @@ const mapStateToProps = state => {
     maxDuration,
     minDuration,
     lookback,
+    traceID: traceIDParams,
```

Comment:
What is the format of the URL here?
`traceID=1&traceID=2` or `traceID=1,2`

If the later, let's use `traceIDs`